bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -188,12 +188,20 @@ spec:
           # As the secret mount is optional we must wait for the files to be present.
           # The service is created in monitor.yaml and this is created in sdn.yaml.
           # If it isn't created there is probably an issue so we want to crashloop.
-          TS=$(curl \
-            --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
-            -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
-            "https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1/namespaces/openshift-sdn/services/sdn" |
-              python -c 'import json,sys; print(json.load(sys.stdin)["metadata"]["creationTimestamp"])'
-          )
+          retries=0
+          until [ "$retries" -ge 20 ]
+          do
```

Comment:
```suggestion
          while [[ "${retries}" -lt 20 ]]; do
```

Reply:
all the stylistic suggestions here still apply, here and elsewhere. (single-line rather than multi-line and `while` rather than `until` because those are both more standard. `[[` rather than `[` and `${retries}` rather than `$retries` because they have fewer gotchas and it matches the kube/OCP shell script style guide that was never finalized...